### PR TITLE
Add automatic testing on Travis

### LINKED ISSUE #2: Add automatic testing (Continuous Integration) on Travis

### PR DESCRIPTION
**Pull request by gaalexandre**
Closes #2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- make test
+- make vet
+- make website-test
+
+matrix:
+  fast_finish: true
+  allow_failures:
+  - go: tip
+env:
+  - GOFLAGS=-mod=vendor GO111MODULE=on
```
Comment (paulrbr-fl) [inline .travis.yml]: I would try to remove the flag `-mod=vendor` to use the new [go 1.11 module system](https://golang.org/cmd/go/#hdr-Modules__module_versions__and_more).

Indeed I really don't like the `vendor/` directory full of 2228 files!

**Review by paulrbr-fl — CHANGES_REQUESTED** [review]
Please check the module dependency management to avoid commiting vendor libraries

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+dist: trusty
+sudo: required
+services:
+- docker
+language: go
+go:
+  - "1.12.x"
```
Comment (paulrbr-fl) [inline .travis.yml:7]: could you also add `1.11` in the list please?

Comment (paulrbr-fl) [reply .travis.yml:7]: perfect thanks!

**Review by paulrbr-fl — APPROVED** [review]
:rocket: great work thanks!

File: scripts/gogetcookie.sh [hunk 3]
```diff
@@ -1,10 +1,9 @@
 #!/bin/bash
 
-touch ~/.gitcookies
-chmod 0600 ~/.gitcookies
+ touch ~/.gitcookies
+ chmod 0600 ~/.gitcookies
```
Comment (fmenou) [inline scripts/gogetcookie.sh:4]: Can you check the whitespaces here and the rest of this file please?

Comment (gaalexandre) [reply scripts/gogetcookie.sh:4]: I will do that.

**Pull request merged**